### PR TITLE
Updated realistic user data to include users with current country != …

### LINKED ISSUE #813: Change birth country for fake users to not match current country
The birth location seems to match the current location exactly for all fake users, but these should deviate at least some of the time

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?

Closes #813 

#### What's this PR do?

Adds 6 users with a current country that is different from their birth country.

Here are the usernames for those 6 users:

['fake.terri.pena',
 'fake.ted.sullivan',
 'fake.camille.jones',
 'fake.ana.cabrera',
 'fake.alvaro.santana',
 'fake.hugo.suarez']

#### How should this be manually tested?

Delete your current set of fake users and re-generate. Check the generated set for the users mentioned above

### CODE REVIEW

File: profiles/management/realistic_program_data.json [hunk 1]
```diff
@@ -1,218 +1,218 @@
 [
```
Comment (gsidebo) [inline profiles/management/realistic_program_data.json]: no actual changes here. this is just a regenerated copy of the same data

**Comment by noisecapella** [comment]
Can you add your generation script too? Maybe create a `tools` directory for that purpose?

**Comment by gsidebo** [comment]
@noisecapella generation script: https://github.com/mitodl/realistic-mm-users

i think we should keep the generation scripts out of this codebase. its extra code that doesn't have anything to do with actual MM functionality. i can add a link in the README if that's something we want

**Comment by noisecapella** [comment]
I'm getting a 404 on that link, can you give me access to it? Is it a private repository?

**Comment by gsidebo** [comment]
oops. sorry about that. just added the ODL engineering team as admins. does it work now?

**Comment by noisecapella** [comment]
Yes, looks good. I feel like this should be public with an open source license, since there's no reason to keep it private (any opinion @pdpinch?)

**Comment by pdpinch** [comment]
Unless there's a good reason, it should be open source. Use the same license (BSD 3 clause?) as micromasters.

**Comment by noisecapella** [comment]
:+1: 

I found a bug when deleting existing users: #818

**Comment by gsidebo** [comment]
script repo has been updated

**Pull request merged**